### PR TITLE
SQL: Standard collection view

### PR DESCRIPTION
**Pull request by unrealwork**
- [x] Added tests to standard collections' view
- [x] Fixed failing tests
- [x] Uniformed test structure

### CODE REVIEW

**Comment by unrealwork** [comment]
**Will fail until #3141 won't be fixed**

**Comment by rmakulov** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlStandardCollectionViewTest.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+     */
+
+    /**
+     * Issue #3126
+     * Test will fail until issue #3141 won't be fixed
+     */
+    @Test
+    public void testAlphabeticalOrder() {
+        String sqlQuery =
+                "SELECT tags, metric.tags, entity.tags, entity.groups FROM '" + TEST_METRIC1_NAME + "'\n" +
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlStandardCollectionViewTest.java]: You can use String.format here

**Comment by KutuzovGregory** [comment]
review completed

File: src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlStandardCollectionViewTest.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    }});
+
+
+    @BeforeClass
+    public static void prepareData() throws Exception {
+        //Series data
+        List<Series> seriesList = new ArrayList<>();
+        seriesList.add(new Series() {{
+            setTags(TAGS);
+            setMetric(TEST_METRIC1_NAME);
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/sql/syntax/SqlStandardCollectionViewTest.java]: Register metric and entity (new Series(entity, metric))

**Comment by Sild** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/sql/response/SqlNullDataTypeTest.java [hunk 3]
```diff
@@ -26,6 +27,9 @@
 
     @BeforeClass
     public static void initialize() throws IOException {
+        Registry.Metric.register(TEST_METRIC1_NAME);
+        Registry.Metric.register(TEST_METRIC2_NAME);
+        Registry.Entity.register(TEST_METRIC1_NAME);
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/response/SqlNullDataTypeTest.java]: entity register for metricName?

**Comment by Sild** [comment]
review done

**Pull request merged**